doc/uftrace-graph.md

```diff
@@ -77,6 +77,9 @@ COMMON OPTIONS
 \--no-sched
 :   Do not show schedule events.
 
+\--no-preempt-sched
+:   Do not show preempt schedule events.
```

Comment:
Please mention that it'd show regular (sleeping) schedule events.  While `--no-sched` will hide both events.

Reply:
I add additional description that "but show regular(sleeping) schedule events" to 
'--no-sched-preempt' option description according to the review.